backend/src/models/Post.ts

```diff
@@ -1,12 +1,28 @@
 import { prop, getModelForClass, modelOptions } from '@typegoose/typegoose';
 import { TagModel } from './Tag';
 
+export enum ContentType {
+  MULTIPLE_CHOICE = 'MULTIPLE_CHOICE',
+  OPEN_RESPONSE_MESSAGE = 'OPEN_RESPONSE_MESSAGE',
+}
+
 export enum PostType {
   BOARD = 'BOARD',
   BUCKET = 'BUCKET',
   LIST = 'LIST',
 }
 
+export class MultipleChoiceOptions {
+  @prop({ required: true })
+  public optionTitle!: string;
+
+  @prop({ required: true })
+  public correct!: boolean;
+
+  @prop({ required: false })
+  public formuala?: boolean;
```

Comment:
Just noticed this typo, we should have this fixed in another PR since I think this code is from an older task.